clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -38,7 +50,24 @@
                     ".cob"
                 ]
             }
-        ]
+        ],
+        "configuration": {
+            "title": "Broadcom COBOL LSP",
+            "properties": {
+                "broadcom-cobol-lsp.cpy-manager.profiles": {
+                    "type": "Object",
+                    "description": "Program name to profile map"
+                },
+                "broadcom-cobol-lsp.cpy-manager.paths": {
```

Comment:
in order to use autocomplete in settings.json we can not combine different properties under one structure to have something like: 
`broadcom-cobol-lsp.cpy-manager: {
                                                           paths: {},
                                                           profiles: {}
}
`
this [link](https://json-schema.org/understanding-json-schema/reference/index.html) is useful for future implementation